src/rest-server/docs/swagger.yaml

```diff
@@ -330,19 +550,30 @@ paths:
           $ref: '#/components/responses/NoUserError'
         '500':
           $ref: '#/components/responses/UnknownError'
-  '/api/v2/user/{user}/group/{group}':
+  '/api/v2/user/{user}/group/':
     parameters:
       - $ref: '#/components/parameters/user'
-      - $ref: '#/components/parameters/group'
     put:
       tags:
         - user
-        - group
       summary: Add a group for user.
-      description: Administrator add a group to other user's grouplist.
+      description: >-
+        Admin only.
+        Basic mode only.
+        Add a group to other user's grouplist.
```

Comment:
only `other user`? An administrator could also add group to himself / herself?

Reply:
Yes, admin can add a group into himself/herslf.